client/gutenberg/extensions/publicize/form.jsx

```diff
@@ -11,8 +11,8 @@
 /**
  * External dependencies
  */
+import isNil from 'lodash/isNil';
```

Comment:
Not really part of this PR but can we just remove the dependence on lodash here 
since the isNil function is just 
```
function isNil(value) {
      return value == null;
}``` 
https://github.com/lodash/lodash/blob/4.17.10/lodash.js#L11972

Reply:
👍Good idea - I'll follow-up in another PR.